crates/db_schema/src/impls/community.rs

```diff
@@ -325,13 +324,15 @@ impl CommunityFollower {
   pub async fn has_local_followers(
```

Comment:
```suggestion
  pub async fn require_local_followers(
```

Reply:
We mostly use names like `check_has_local_followers`

Reply:
K, I'll change that name shortly.